<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MLPsim</class>
 <widget class="QWidget" name="MLPsim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>150</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Regresi dengan Multilayer Perceptron</string>
  </property>
  <widget class="QPushButton" name="pbOpen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>104</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buka Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSim">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>104</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulasi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>104</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConfig">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>104</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Konfigurasi MLP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
